README.md

```diff
@@ -89,15 +89,15 @@ Airflow is not a streaming solution, but it is often used to process real-time d
 
 Apache Airflow is tested with:
 
-|             | Main version (dev)           | Stable version (2.7.1) |
-|-------------|------------------------------|------------------------|
-| Python      | 3.8, 3.9, 3.10, 3.11         | 3.8, 3.9, 3.10, 3.11   |
-| Platform    | AMD64/ARM64(\*)              | AMD64/ARM64(\*)        |
-| Kubernetes  | 1.24, 1.25, 1.26, 1.27, 1.28 | 1.24, 1.25, 1.26, 1.27 |
-| PostgreSQL  | 11, 12, 13, 14, 15, 16       | 11, 12, 13, 14, 15     |
-| MySQL       | 8.0, 8.1                     | 5.7, 8.0               |
-| SQLite      | 3.15.0+                      | 3.15.0+                |
-| MSSQL       | 2017(\*\*), 2019(\*\*)       | 2017(\*\*), 2019(\*\*) |
+|             | Main version (dev)           | Stable version (2.7.2)       |
+|-------------|------------------------------|------------------------------|
+| Python      | 3.8, 3.9, 3.10, 3.11         | 3.8, 3.9, 3.10, 3.11         |
+| Platform    | AMD64/ARM64(\*)              | AMD64/ARM64(\*)              |
+| Kubernetes  | 1.24, 1.25, 1.26, 1.27, 1.28 | 1.24, 1.25, 1.26, 1.27, 1.28 |
```

Comment:
I'm not sure about K8S, but usually we add/remove support on feature realise , e.g. 2.8.0
However it could be not applicable to K8S